create_db.py

```diff
@@ -135,6 +135,36 @@ def insert_conversations(session):
     return line_to_conv_mapping
 
 
+def clean_records(session):
+    real_ids = {'u5784': 'u5783', 'u5786': 'u5785', 'u6564': 'u6563'}
+
+    for dup_char_id in ('u5784', 'u5786', 'u6564'):
```

Comment:
You could call [`dict.keys`](https://devdocs.io/python~3.6/library/stdtypes#dict.keys) on `real_ids` here, ensuring that you do not have to hardcode these values twice.